### PR TITLE
Add protocol to feature service host

### PR DESCRIPTION
**Pull request by kevinhartman**

### CODE REVIEW

File: ops/create-cluster.sh [hunk 1]
```diff
@@ -55,7 +55,7 @@ done
 readonly graphql_service_host="fortis-services.${DEIS_ROUTER_HOST_ROOT}.nip.io"
 readonly fortis_interface_host="fortis-interface.${DEIS_ROUTER_HOST_ROOT}.nip.io"
 readonly feature_service_db_conn_str="${FEATURE_SERVICE_DB_CONNECTION_STRING}"
-readonly feature_service_host="feature-service.${DEIS_ROUTER_HOST_ROOT}.nip.io"
+readonly feature_service_host="http://feature-service.${DEIS_ROUTER_HOST_ROOT}.nip.io"
```
Comment (Smarker) [inline ops/create-cluster.sh:58]: should this instead be `readonly feature_service_url="http://feature-service.${DEIS_ROUTER_HOST_ROOT}.nip.io" `?

**Review by Smarker — APPROVED** [review]
LGTM - with comment

File: ops/create-cluster.sh (hunk 1 above)
Comment (kevinhartman) [reply ops/create-cluster.sh:58]: This is currently working (I see this value in the right env var), it's just that it was missing the protocol. Is there some issue you see here?

Comment (Smarker) [reply ops/create-cluster.sh:58]: No, I just was referring to the name. (I don't think hosts typically have `http://` in front that's all.

Comment (kevinhartman) [reply ops/create-cluster.sh:58]: I would agree, though I believe the term "host" is used in referring to this from across multiple repos, so if someone feels strongly about it, they (or you :) ) can address it in a separate PR(s). Though I wouldn't prioritize this over anything else right now.

Comment (Smarker) [reply ops/create-cluster.sh:58]: ok true yea I think holding off on this change would be best

**Pull request merged**